library/ui-strings/src/main/res/values-fr/strings.xml

```diff
@@ -2599,7 +2599,7 @@
     <string name="location_share_loading_map_error">Impossible de charger la carte
 \nCe serveur d’accueil n’a peut-être pas été configuré pour afficher les cartes.</string>
     <string name="a11y_open_settings">Ouvrir les paramètres</string>
-    <string name="all_chats">Toutes les conversations</string>
+    <string name="all_chats">Accueil</string> <!-- Tchap: change wording title -->
```

Comment:
We could have reused the existing `group_details_home` string in the layout, we would also have benefited from the English translation which has not changed in your PR. Otherwise, the meaning of the word here is not the same, so it would have been better to have a new key in Tchap, if you don't want to reuse the key that I mentioned (`all tchats` can be used in several places in the project and I am not sure we want to change the meaning for all potential places).